lib/app/modules/core/capability_statement_sequence.rb

```diff
@@ -71,9 +71,22 @@ class CapabilityStatementSequence < SequenceBase
         end
       end
 
-      test 'FHIR server supports the conformance interaction that defines how it supports resources' do
+      test 'Test FHIR version' do
```

Comment:
Let's try and be more explicit about what `Test FHIR version` really means.
```suggestion
      test 'FHIR version of the server matches the FHIR version expected by tests' do
```